wildcard.json

```diff
@@ -59,6 +59,7 @@
   "jto.kr",
   "justemail.ml",
   "kadokawa.top",
+  "knowledgemd.com",
```

Comment:
We don't use this file. So you will add more logic in user-acl-two to do the wildcard match. If we are fine for just the domain check then adding the domain to index.json is fine.

Also, bump up the package version ?

Reply:
i bumped up the version